snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-0

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for daily stats ping, tracking instance metrics and settings",
+    "self": {
+      "vendor": "com.metabase",
+      "name": "instance_stats",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "instance_attributes": {
+        "description": "Key-value pairs of instance attributes",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "description": "A single instance attribute",
+          "properties": {
+            "key": {
+              "description": "The key for this attribute",
+              "type": "string",
+              "maxLength": 255
+            },
+            "value": {
+              "description": "The value of this attribute",
+              "type": ["string", "boolean", "integer", "null"],
+              "maxLength": 255,
+              "minimum": 0,
+              "maximum": 2147483647
+            }
+          }
+        }
+      },
+      "features": {
+        "description": "Features",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "description": "A single instance attribute",
+          "properties": {
+            "name": {
+              "description": "The unique name of the feature",
+              "type": "string",
+              "maxLength": 255
+            },
+            "available": {
+              "description": "Whether the feature is available, i.e. can it be enabled/disabled or is it always on",
+              "type": "boolean"
+            },
+            "enabled": {
+              "description": "Whether the feature is enabled, i.e. can it be used by the users/instance",
+              "type": "boolean"
+            }
+          },
+          "required": [
+            "name",
+          "available",
+          "enabled"
+          ],
+          "additionalProperties": true
+        }
+      },
+      "metadata": {
+        "description": "Metadata about the anonymous stats collection",
+        "type": "object",
+        "additionalProperties": {
+          "description": "Individual metadataproperties",
+          "type": ["string", "integer"],
+          "maxLength": 255,
+          "minimum": 0,
+          "maximum": 2147483647
+        }
```

Comment:
**thought:** Do we want to limit this to key-value pairs as well? Would make automatically processing it downstream data land easier. Otherwise, we might need to manually process individually added key, values (especially if someone ends up adding JSON values or something).

Reply:
As in an array of objects, where each object has `"key"` and `"value"` (or similar) keys? Sure, I can change that if it would be better for you. Just let me know if that's what you had in mind.